department-of-veterans-affairs/caseflow
# Connecting daily docket to backend

Resolves #7073 

### Description
This PR connects the daily docket to the backend. Edit/save functionality will come in a future PR.

### Testing Plan
1. Find the id for a central office hearing
1. Navigate to /hearings/schedule/docket/${id}
1. Confirm page renders correctly
1. Create a child record for a video master record
1. Navigate to /hearings/schedule/docket/${master_record_id}
1. Confirm page renders correctly
